### PR TITLE
GUVNOR-3462: [Guided Decision Table] Descriptions of each step in the Wizard

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3462

### CODE REVIEW

**Comment by manstis** [comment]
See https://www.youtube.com/watch?v=Jza3VF0L8vE&feature=youtu.be

**Comment by manstis** [comment]
@cristianonicolai Looks like the [IT Order](https://github.com/kiegroup/kie-wb-playground/tree/master/itorders) example is failing to compile on Jenkins.

**Comment by manstis** [comment]
```
19:13:58 [ERROR] Unable to build KieBaseModel:defaultKieBase
19:13:58 Process Compilation error : Process org.jbpm.demo.itorders.Order for IT hardware(itorders.orderhardware)
19:13:58 	org/jbpm/demo/itorders/Process_org$u46$jbpm$u46$demo$u46$itorders$u46$Order_for_IT_hardware1497444760.java (9:487) : The method isTrue(Boolean) in the type KieFunctions is not applicable for the arguments (Object)
19:13:58 	org/jbpm/demo/itorders/Process_org$u46$jbpm$u46$demo$u46$itorders$u46$Order_for_IT_hardware1497444760.java (15:739) : The method isFalse(Boolean) in the type KieFunctions is not applicable for the arguments (Object)
```

**Comment by manstis** [comment]
Jenkins please retest this (should the failure be transient)

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties [hunk 1]
```diff
@@ -78,44 +83,76 @@ AttributeColumnPageView.Salience=salience
 AttributeColumnPageView.SalienceDescription=Salience is a form of priority where rules with higher salience values are given higher priority when ordered in the Activation queue.
 AttributeColumnPageView.Timer=timer
 AttributeColumnPageView.TimerDescription=Timer attribute.
+AttributeColumnPageView.AttributeColumnPageDescription=Add one or more attribute columns representing any of the DRL rule attributes, such as Saliance, Enabled, Date-Effective, and others.
```
Comment (karreiro) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties]: I think that it's typo. It's "Salience", right?

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties [hunk 2]
```diff
@@ -20,7 +20,11 @@ EditMenu.paste=Paste
 EditMenu.deleteCells=Delete cell(s)
 EditMenu.deleteColumns=Delete column(s)
 EditMenu.deleteRows=Delete row(s)
-EditMenu.otherwise=Otherwise
+EditMenu.otherwise=Insert "any other" value
+EditMenu.otherwiseDescription=<p>For condition columns in Guided Decision Tables, you can apply an "any other" value to table cells within the column if the following parameters are set: \
```
Comment (karreiro) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties]: I have mixed feelings about putting HTML tags here. But, I think that it's the best approach in this context.

An alternative would be the use of something like this:
```
EditMenu.otherwiseDescription1=...
EditMenu.otherwiseDescription2=...
EditMenu.otherwiseDescription3=...
```
I think that this alternative pollutes the code of the View.

So, good approach :-)

**Comment by manstis** [comment]
@karreiro What about a hybrid, where the page title and info box are merged.

https://www.youtube.com/watch?v=-prZZanoGHM&feature=youtu.be

hybrid-1
hybrid-2

**Comment by karreiro** [comment]
@manstis great idea 👏 :-)

**Comment by manstis** [comment]
@karreiro @jomarko I've implemented the latter _hybrid_ option in a new commit.

**Pull request merged**